### DISCUSSION TITLE
Possible way to skip the scratch buffer while exiting?

### DISCUSSION THREAD

**Discussion by kajuburfi** [helix-editor/helix#2515] [Q&A]
I have bound <kbd>Space</kbd><kbd>q</kbd> to <kbd>:buffer-close</kbd>. But If I have only 1 buffer(file) left, it goes to scratch buffer.

My question is: Is it possible to skip this scratch buffer when I press <kbd>Space</kbd><kbd>q</kbd>? If so, how?

**Comment by peterjc** [comment]
I think you're asking for the behavior I expected from ``:buffer-close`` aka ``:bc`` aka ``:bclose`` - when you have a single buffer open which has no unsaved changes, this would close the buffer and quit helix.

(In my mind this is like multiple window or multiple tab applications - if you close the final tab/window, the application usually closes)
